cms/migrations/0023_auto_20180719_1638.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.14 on 2018-07-19 16:38
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('cms', '0022_auto_20180620_1551'),
```

Comment:
contenttypes should be added as dependency here

Reply:
Krzysztof asked me to remove it and have a purely linear dependency flow.  I will put it back in if you say.

Reply:
I only talked about cms migrations depending on each other in linear fashion. This migration uses contenttypes, so it also needed to be in the dependency list.

Reply:
Sorry, I misunderstood you then, since 0010_migrate_use_structure.py had a dependency on contenttypes.